internal/coreinternal/attraction/attraction.go

```diff
@@ -172,7 +172,7 @@ type AttrProc struct {
 // and returns a AttrProc to be used to process attributes.
 // An error is returned if there are any invalid inputs.
 func NewAttrProc(settings *Settings) (*AttrProc, error) {
-	var attributeActions []attributeAction
+	attributeActions := make([]attributeAction, len(settings.Actions))
```

Comment:
Why not 
```suggestion
	attributeActions := make([]attributeAction, 0, len(settings.Actions))
```

as below? IMO we should pick one style and stick with it

Reply:
sure, done!